src/kbmod/analysis_utils.py

```diff
@@ -71,36 +74,53 @@ def load_images(
         print("Loading Images")
         print("---------------------------------------")
 
+        # Get the bounds for the characters to use for the visit ID in the file name.
+        id_start = visit_in_filename[0]
+        id_end = visit_in_filename[1]
+        id_len = id_end - id_start
+
         # Load a mapping from visit numbers to the visit times. This dictionary stays
         # empty if no time file is specified.
         image_time_dict = OrderedDict()
         if time_file:
             visit_nums, visit_times = np.genfromtxt(time_file, unpack=True)
             for visit_num, visit_time in zip(visit_nums, visit_times):
-                image_time_dict[str(int(visit_num))] = visit_time
+                visit_num_str = f"{int(visit_num):0{id_len}}"
+                image_time_dict[visit_num_str] = visit_time
+        if verbose:
+            print(f"Loaded {len(image_time_dict)} time stamps.")
 
         # Load a mapping from visit numbers to PSFs. This dictionary stays
         # empty if no time file is specified.
         image_psf_dict = OrderedDict()
         if psf_file:
             visit_nums, psf_vals = np.genfromtxt(psf_file, unpack=True)
             for visit_num, visit_psf in zip(visit_nums, psf_vals):
-                image_psf_dict[str(int(visit_num))] = visit_psf
+                visit_num_str = f"{int(visit_num):0{id_len}}"
```

Comment:
So does this spit out a string (e.g. for the 4th of 120) like "4:0120"? Or is there some extra fanciness happening here? If this is a format that is expected elsewhere, could it be constructed in a (documented) helper method?

Reply:
It spits out a zero padded string based on the integer `visit_num` where `id_len` is the length of the string. Since we allow the user to set the starting and ending indices for the id (and thus use variable length ids) we need the extra level of indirection when zero padding it.

Reply:
That *is* fancy!